app/src/components/governance/ProposalDescription.vue

```diff
@@ -11,8 +11,6 @@
             "
           >
           {{ proposal.description }}
-          <p class="parameter">New Parameter: {{ proposal.changes }}</p>
-          <p v-if="proposal.beneficiary">Beneficiary: {{ proposal.beneficiary.address }}</p>
         </pre>
           <p v-else>{{ proposal.description }}</p>
```

Comment:
we can remove this v-else because now we will always display `proposal.description`

Reply:
ah true! Good eye

Reply:
ah no, but the difference is either we show the robotic version or the normal version, you know what I mean

Reply:
Robotic version, our friend @mariopino asking for a grant:

![image](https://user-images.githubusercontent.com/40721795/93909286-e9f18100-fcff-11ea-931f-736f68e8f31c.png)

Normal version (not treasury and not Parameter change. Usually Text):

![image](https://user-images.githubusercontent.com/40721795/93909405-0392c880-fd00-11ea-8374-5b02b4ce1281.png)

Reply:
So it is better to keep it like this